src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/ContactsServiceTest.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service
+
+import org.mockito.Mockito.mock
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.BookingContactsRepository
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.ContactsRepository
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.VideoBookingRepository
+
+class ContactsServiceTest {
```

Comment:
Can copy across the old BookingContactsService tests here too? Then add the contact (no booking) stuff?